src/core/jwt.py

```diff
@@ -3,6 +3,7 @@
 from jose import jwt, JWTError
 from fastapi import Response
 from sqlalchemy.ext.asyncio import AsyncSession
+import sys
```

Comment:
Thank you for your support. Import order is not compatible with PEP8 rules. If you can fix it, I will merge it.
1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.
You can learn more: [https://peps.python.org/pep-0008/#imports](https://peps.python.org/pep-0008/#imports)

Reply:
Updated import statement to support pep-0008 rules